.drone.yml

```diff
@@ -58,6 +58,63 @@ steps:
       - 2-2-stable
       - 2-3-stable
 
+- name: docs-generate
+  image: quay.io/openware/sdk-tools:test-fetch1
+  environment:
+    BOT_USERNAME: kite-bot
+    BOT_NAME: Kite Bot
+    BOT_EMAIL: kite-bot@heliostech.fr
+    REPO_NAME: ${DRONE_REPO}
+    BRANCH_NAME: ${DRONE_BRANCH}
+    SDK_BRANCH: test
+    GITHUB_API_KEY:
+      from_secret: kite_bot_key
+  commands:
+    - cd /sdk
+    - bundle exec rake ci:pull
+    - bundle exec rake ci:changelog['/drone/src/docs']
+    - bundle exec rake ci:copy_docs['/drone/src/docs']
+    - bundle exec rake ci:push
+  when:
+    event:
+      - push
+    branch:
+      - master
+      - 2-0-stable
```

Comment:
Please, check stable branches on barong. We don't have 2-0-stable branch on barong